about.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" type="text/css" href="./styles/mainStyle.css">
+  <link rel="stylesheet" type="text/css" href="./styles/mobileNav.css">
+  <link rel="stylesheet" type="text/css" href="./styles/aboutStyle.css">
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="navigator-menu">
+      <img class="navigator-menu__burger" src="./images/icons/burgerIcon.png"> 
+    </nav>
+
+    <!-- nav menu -->
+
+    <div class="hidden-menu">
+      <ul class="hidden-menu__ul">
+        <li class="hidden-menu__li x-container"><img class="hidden-menu__x-element" src="./images/xIcon.png"></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option" href="./index.html">Home</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Buy</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Sell</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Sponsor</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">News</a></li>
+        <li class="hidden-menu__li"><button type="submit" class="hidden-menu__button">Campain</button></li>
+      </ul>
+    </div>
+
+    <!-- Second nav bar -->
+
+    <div class="nav-bar-container">
+      <div class="nav-bar">
+        <div class="nav-bar__icons"><img src="./images/facebook.png"></div>
+        <div class="nav-bar__icons"><img src="./images/twitter.png"></div>
+        <div class="nav-bar__language"><p class="nav-bar__english">English</p></div>
+      </div>
+      <div class="desktop-menu-container">
+        <ul class="desktop-menu">
+          <li class="desktop-menu__icon-container"><img class="desktop-menu__icon" src="./images/icons/mainIcon.png"></li>
```

Comment:
- [ ] The logo from your `About` page should link to your homepage.
- Kindly make do to refactor it to be a link tag, so the navigation becomes easier 👍 

## Requiremets 👍 
![image](https://github.com/EvansSnave/capstone-project/assets/50990925/e759cd70-6a1c-4125-8482-764ac941e44d)